.github/workflows/main.yaml

```diff
@@ -0,0 +1,12 @@
+name: CI
```

Comment:
@pmalirz `CI` is a pretty generic name for this GitHub Action.

Do you want to add further Ci jobs to this file?
Or if you want to keep this focused on awesome lint, then we could change it to `name: Awesome Lint` possibly.

But your call, I have only used GitHub Actions a little bit, so I don't know what works best here :)

Reply:
@spier this config was copied from the reference / sugested awesome-lint repository... it's been working fin on my local and on the branch so I left it in that shape... of course I'm happy to tune it as per your suggestions, not a problem.

Reply:
Yeah, I didn't study github actions as well

Reply:
Let's keep it as is then, and change it in the future if we see a need.